src/app/tabs.component.js

```diff
@@ -0,0 +1,19 @@
+export const tabsComponent = {
+  selector: 'appTabs',
+  templateUrl: '/tabs.component.html',
+  controller: TabsCtrl,
+};
+
+class TabsCtrl {
```

Comment:
This is not right. Classes are not hoisted in the same way that functions do. As a result, `controller: TabsCtrl` is the same as `controller: undefined`.

You should either move the class definition above the `tabsComponent` object initialization or define the class inline (as you do with `VersionStampCtrl`):

```suggestion
  controller: class TabsCtrl {
    ...
  },
};
```